doc/compiler/misc/BestPractices.md

```diff
@@ -1,5 +1,5 @@
 <!--
-Copyright (c) 2019, 2020 IBM Corp. and others
+Copyright (c) 2019, 2022 IBM Corp. and others
```

Comment:
I think the copyright checker will complain because it just checks for what files were affected in a PR. Also technically the copyrights of all files should be changed every year, but we change it on an as needed basis to prevent a yearly massive CR change.